src/js/components/Search.jsx

```diff
@@ -173,11 +174,35 @@ class Search extends Component {
         </div>
       )
     }
+    
+    if (searchResults.error) {
+      const {message} = searchResults.error
+      const spanStyle = {...style.span, color: 'red'}
+      let text = {}
+       
+      try {
```

Comment:
This was so REST could send an error in any format. Sometimes we want the error formatted to display to a user. For example:
You do not have the correct LACs/COIs to view this data:
LACs: 123_456
COIs: ABC_DEF 

A string does not allow this.